test/test-cases/hello-world/sku.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  target: 'actual-output'
```

Comment:
I'd probably leave this as the default, build to `dist`, since this is normal sku behaviour. Shouldn't be hard to make our test setup use this folder structure.